main.js

```diff
@@ -1,163 +1,291 @@
+        name: 'HD TV',
+        brand: 'Toshiba',
+        price: 161,
+        availableSizes: [32],
+        refreshRate: 50,
+        screenType: 'LED',
+        screenQuality: 'Full HD',
+        smartTv: false,
+        options: {
+            wifi: false,
+            speech: false,
+            hdr: true,
+            bluetooth: false,
+            ambiLight: false,
+        },
+        originalStock: 10,
+        sold: 8,
     },
-    originalStock: 10,
-    sold: 8,
-  },
-];
\ No newline at end of file
+];
+
+console.log("//////////////// Opdracht 1a ////////////////")
+
+inventory.map((tvType) => {
+    console.log(tvType.type);
+});
+
+console.log("//////////////// Opdracht 1b ////////////////")
+
+const soldOutTvs = inventory.filter((soldOutTv) => {
+    return soldOutTv.sold < soldOutTv.originalStock;
+});
+console.log(soldOutTvs);
+
+console.log("//////////////// Opdracht 1c ////////////////")
+
+const ambiLightFinder = inventory.filter((ambiLightFind) => {
+    return ambiLightFind.options.ambiLight === true;
+})
+
+console.log(ambiLightFinder);
+
+console.log("//////////////// Opdracht 1d ////////////////")
+
+const tvLowToHigh = inventory.sort((a, b) => {
+    return a.price - b.price;
+})
+
+console.log(tvLowToHigh);
+
+
+console.log("//////////////// Opdracht 2a ////////////////")
+
+let soldTVs = 0;
+for (let i = 0; i < inventory.length; i++) {
+    soldTVs = (soldTVs + inventory[i].sold);
+}
+
+console.log(soldTVs)
+
+console.log("//////////////// Opdracht 2b ////////////////")
+
+const numberOfSoldTvs = document.getElementById("SoldTvs");
+numberOfSoldTvs.textContent = soldTVs
+
+console.log("//////////////// Opdracht 2c ////////////////")
+
+let boughtInTvs = 0;
+for (let i = 0; i < inventory.length; i++) {
+    boughtInTvs = (boughtInTvs + inventory[i].originalStock);
+}
+console.log(boughtInTvs);
+
+console.log("//////////////// Opdracht 2d ////////////////");
+
+const numberOfBoughtInTvs = document.getElementById("boughtInTvs");
+numberOfBoughtInTvs.textContent = boughtInTvs
+
+console.log("//////////////// Opdracht 2e ////////////////");
+
+const inStock = boughtInTvs - soldTVs;
+const numberOfTvInStock = document.getElementById("stockTvs");
+numberOfTvInStock.textContent = inStock;
+
+
+console.log("//////////////// Opdracht 3a ////////////////");
+
+const brandsList = inventory.map((brand) => {
+    return " " + brand.brand
+});
+console.log(brandsList)
+
+const brandsInStore = document.getElementById("brandList");
+brandsInStore.textContent = brandsList;
+
+
+console.log("//////////////// Opdracht 3b ////////////////");
+
+function brandNames(array) {
+    let outcome = array.map((brand) => {
+        return brand.brand
+    })
+    return outcome
+}
+
+const brands = brandNames(inventory);
+console.log(brands);
+
+
+console.log("//////////////// Opdracht 4a ////////////////");
+
+function nameString(array, indexnumber) {
+    let name = array.map((id) => {
+        return id.brand + " " + id.type + " - " + id.name
```

Comment:
Ik vind dit altijd net wat lekkerder lezen
```suggestion
        return `${id.brand} ${id.type} - ${id.name}`;
```